yarn.lock

```diff
@@ -2,16 +2,17 @@
 # yarn lockfile v1
 
 
-"@authenio/xml-encryption@^0.11.3":
-  version "0.11.3"
-  resolved "https://registry.yarnpkg.com/@authenio/xml-encryption/-/xml-encryption-0.11.3.tgz#35379a8a87bbed3f34907a0359e8e1c30ce24bdd"
-  integrity sha512-zFLGBw/iNmLq5Udg+OJfsuERsTjcfvWtP6ZzL55EVwGu3bYsxuq1ARRNfNLHCXv4cNP49o8nLwpgA6VmXzRp/w==
-  dependencies:
-    async "^2.1.5"
-    ejs "^2.5.6"
-    node-forge "^0.7.4"
-    xmldom "~0.1.15"
-    xpath "0.0.27"
+"@authenio/samlify-xsd-schema-validator@^1.0.3":
```

Comment:
Please also update the yarn.lock, we completely separate the validator modules from samlify. You can remove node_modules and run yarn again (just to have a clean install) after you update package.json, then this lock file will be updated automatically.

The script used by Travis CI will do the test with validator modules as well.

Reply:
Yes but tests don't pass locally without it :) Please see my other reply.

Reply:
Refer to the concern that I have mentioned above :)

Reply:
Removed :)